docs/DeveloperGuide.md

```diff
@@ -59,6 +60,22 @@ The rest of the App consists of four components.
 * [**`Storage`**](): Reads data from, and writes data to, the hard disk.
 
 ## Implementations
+
+### Ui Class
+
+The Ui class is represented by the image below. The Ui class handles user input and
+displaying of relevant information including error messages for the program. In the diagram below,
```

Comment:
Perhaps "the displaying of relevant information...." would be a better phrasing?